test/metabase/native_query_analyzer_test.clj

```diff
@@ -39,13 +38,8 @@
         (is (= {:explicit #{(mt/id :venues :id)} :implicit nil}
                (q "select id from venues"))))
       (testing "quotes stop case matching"
-        ;; MySQL does case-insensitive string comparisons by default; quoting does not make it consider case
-        ;; in field names either, so it's consistent behavior
```

Comment:
This is not true anymore for field names, because we changed the collation to be case-sensitive in this migration:
https://github.com/metabase/metabase/pull/44866/files#diff-e6ec4fc8084d80ae2ebedde98c98f358ec7ac526de5c339ebcb454cbda62b221R6490